selenium_tests/cross_listing/remove_cross_listing_tests.py

```diff
@@ -0,0 +1,56 @@
+from ddt import ddt, data, unpack
+
+from selenium_common.base_test_case import get_xl_data
+
+from selenium_tests.cross_listing.cross_listing_base_test_case import \
+    CrossListingBaseTestCase, TEST_DATA_CROSS_LISTING_MAPPINGS
```

Comment:
Will punt this one for a future round. The base test case class for cross listing looks pretty clean, and it's clear where the test data is coming from.  The benefit is true as you stated and the add tests could make use for that as well.  For this round, let's keep as is - it currently works with same effect.